apps/institution-service/src/controllers/permissionController.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response } from "express";
+import { jwtDecode } from "jwt-decode";
+import { UiUserPermissions } from "@repo/shared-utils";
+
+interface DecodedToken {
+  permissions: string[];
+}
+
+export const getPermissions = (req: Request, res: Response) => {
+  const token = req.headers.authorization?.split(" ")?.[1];
+  const decodedToken = jwtDecode<DecodedToken>(token as string);
+
+  const permissions = {
+    canCreateInstitution: !!decodedToken?.permissions?.find((permission) =>
+      [
+        UiUserPermissions.CREATE_INSTITUTION,
+        UiUserPermissions.CREATE_INSTITUTION_AGGREGATOR,
+      ].includes(permission),
+    ),
+  };
```

Comment:
`some` returns true if any of the tests pass and I feel like it makes the code a little easier to read, what do you think of this change?

```suggestion
  const permissions = {
      canCreateInstitution: [
        UiUserPermissions.CREATE_INSTITUTION,
        UiUserPermissions.CREATE_INSTITUTION_AGGREGATOR,
      ].some((permission) => decodedToken?.permissions?.includes(permission)),
    };
```

Reply:
👍